pkg/broker/broker.go

```diff
@@ -32,11 +32,20 @@ import (
 
 // ProvisionParameters contains the provision parameter fields
 type ProvisionParameters struct {
-	RepositoryName  string `json:"repository_name"`
-	RepositoryOwner string `json:"repository_owner"`
-	MigrateURL      string `json:"migrate_url,omitempty"`
+	RepositoryName     string             `json:"repository_name"`
+	RepositoryOwner    string             `json:"repository_owner"`
+	MigrateURL         string             `json:"migrate_url,omitempty"`
+	OrganizationPolicy organizationPolicy `json:"organization_policy,omitempty"`
```

Comment:
Let's move this to a start-up option, which defaults to not having the side effect.